cumulusci/tasks/bulkdata/load.py

```diff
@@ -654,14 +658,26 @@ def _set_viewed(self):
                 object_names.add(object_name)
         # collect SobjectName that have custom tabs
         if custom_objects:
-            for record in self.sf.query_all(
-                "SELECT SObjectName FROM TabDefinition WHERE IsCustom = true AND SObjectName IN ('{}')".format(
-                    "','".join(sorted(custom_objects))
+            try:
+                custom_tab_objects = self.sf.query_all(
+                    "SELECT SObjectName FROM TabDefinition WHERE IsCustom = true AND SObjectName IN ('{}')".format(
+                        "','".join(sorted(custom_objects))
+                    )
                 )
-            )["records"]:
-                object_names.add(record["SobjectName"])
-
-        for mapped_item in sorted(object_names):
-            self.sf.query_all(
-                f"SELECT Id FROM {mapped_item} ORDER BY CreatedDate DESC LIMIT 1000 FOR VIEW"
-            )
+                for record in custom_tab_objects["records"]:
+                    object_names.add(record["SobjectName"])
+            except Exception:
+                self.logger.warning(
+                    "Cannot get the list of custom tabs to set recently viewed status on them."
+                )
+        with get_org_schema(self.sf, self.org_config) as org_schema:
+            for mapped_item in sorted(object_names):
+                if org_schema[mapped_item].mruEnabled:
+                    try:
+                        self.sf.query_all(
+                            f"SELECT Id FROM {mapped_item} ORDER BY CreatedDate DESC LIMIT 1000 FOR VIEW"
+                        )
+                    except Exception:
+                        self.logger.warning(
+                            f"Cannot set recently viewed status for {mapped_item}"
+                        )
```

Comment:
Are you intentionally hiding details about why not?

Reply:
4f5ff9bd3